packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -25,6 +29,10 @@ class PreferencesController extends DisposableController
   ValueListenable<bool> get vmDeveloperModeEnabled => _vmDeveloperMode;
   final _vmDeveloperMode = ValueNotifier<bool>(false);
 
+  ValueListenable<bool> get verboseLoggingEnabled => _verboseLogging;
+  final _verboseLogging =
+      ValueNotifier<bool>(Logger.root.level == Level.FINEST);
```

Comment:
store `Level.FINEST` in a const `verboseLoggingLevel` and use here as well as below where Level.FINEST is used a second time. This will guarantee these stay in sync